database/schema.sql

```diff
@@ -207,6 +207,72 @@ CREATE TABLE rotation_move_history (
   FOREIGN KEY (new_bucket) REFERENCES rotation_buckets (bucket_id)
 );
 
+-- BUDGET TABLES
+
+-- Table of all accounts Ruddock owns.
```

Comment:
Can you give an example in the comments of what would go in here? I'm not sure if this would be "President" or "President 2016-2017" conceptually, for example.